crates/libcontainer/src/container/builder.rs

```diff
@@ -225,6 +261,27 @@ mod tests {
             .context("build container")?;
         assert_eq!(path_builder.pid_file, Some(cwd.join("not/existing/path")));
 
+        // validate container_id
+        let result =
```

Comment:
These tests should be in their own testcases.

Reply:
done